diff --git a/drivers/net/ethernet/microchip/enc28j60_hw.h b/drivers/net/ethernet/microchip/enc28j60_hw.h
index 25b41de..2bb2179 100644
--- a/drivers/net/ethernet/microchip/enc28j60_hw.h
+++ b/drivers/net/ethernet/microchip/enc28j60_hw.h
@@ -304,6 +304,6 @@
 #define MAX_FRAMELEN		1518
 
 /* Preferred half duplex: LEDA: Link status LEDB: Rx/Tx activity */
-#define ENC28J60_LAMPS_MODE	0x3476
+#define ENC28J60_LAMPS_MODE	0x3C12
 
 #endif
